torch/fx/_symbolic_trace.py

```diff
@@ -1018,7 +1010,7 @@ def _autowrap_check(
 
 
 @compatibility(is_backward_compatible=True)
-def wrap(fn_or_name: Union[str, Callable]):
+def wrap(fn_or_name: Union[str, Callable], visible_to_make_fx=False):
```

Comment:
Needs some tests to demonstrate the use of `visible_to_make_fx`.

Reply:
Added example usage including decorator usage.